packages/site/src/data/vue/podcasts.ts

```diff
@@ -1,5 +1,87 @@
 import { Podcast } from "@framework/system/src/models/podcast"
 
-export const podcastTags = [] as const
+export const podcastTags = ["general", "graphQL", "vuejs", "quasar"] as const
 
-export const podcasts: Podcast<typeof podcastTags[number]>[] = []
+export const podcasts: Podcast<typeof podcastTags[number]>[] = [
+	{
+		title: "Modern Web",
+		image:
+			"https://pbcdn1.podbean.com/imglogo/image-logo/984467/modern_web_9bpnd.jpg",
+		hosts: ["ThisDot Labs"],
+		description:
+			"Modern Web is a podcast that explores next generation frameworks, standards, and techniques.",
+		rss: "https://www.podbean.com/site/podcatcher/index/blog/7YqKYcoGcvP",
+		href: "https://modernweb.podbean.com/",
+		tags: ["general"],
+	},
+	{
+		title: "Views on Vue",
+		image:
+			"https://assets.fireside.fm/file/fireside-images/podcasts/images/4/4b532b8f-dc99-4b48-90f1-023bd6c0cd90/cover_medium.jpg?v=0",
+		hosts: [
+			"Lindsay Wardell",
+			"Luke Diebold",
+			"Solomon Eseme",
+			"Steve Edwards",
+			"Charles Wood",
+		],
+		description:
+			"A weekly discussion among Vue developers about Vue and it's ecosystem.",
+		rss: "",
+		href: "https://viewsonvue.com/",
+		tags: ["vuejs"],
+	},
+	{
+		title: "The Official Vue News",
+		image:
+			"https://is1-ssl.mzstatic.com/image/thumb/Podcasts115/v4/9f/0f/44/9f0f44f5-2478-96a8-af41-91d0fd87c9f6/mza_8437862612976381859.jpg/313x0w.webp",
+		hosts: ["Vue Mastery"],
+		description:
+			"We cover the latest Vue news and tutorials in about 5 minutes. Our goal is to give you the information you need to stay on top of your game as a Vue developer. Links to all the stories we talk about can be found in our weekly newsletter and on http://news.vuejs.org",
```

Comment:
Descriptions shouldn't be in first person (we/our, etc.) since we're not the podcast's website. They should also not include links.

Reply:
Roger that